lineman/app/services/thread_query_service.coffee

```diff
@@ -52,13 +54,16 @@ angular.module('loomioApp').factory 'ThreadQueryService', (Records) ->
           view.applyFind(lastActivityAt: { $gt: moment().startOf('day').subtract(6, 'week').toDate() })
           view.applyWhere (thread) -> thread.isUnread()
 
-      if _.contains(filters, 'show_muted')
-        view.applyFind(volume: { $eq: 'mute' })
-      else
-        view.applyFind(volume: { $ne: 'mute' })
-
       _.each filters, (filter) ->
         switch filter
+          when 'show_muted'
+            view.applyFind($or: [{volume:  { $eq: 'mute' }},
```

Comment:
@gdpelican I think we need to merge membership onto group in serialisation. The problem being that mutedGroupIds is going to be frozen at view creation.. and we'd rather just say group.volume == 'mute' anyway